nemo/collections/common/tokenizers/en_ja_tokenizers.py

```diff
@@ -15,7 +15,6 @@
 from typing import List
 
 from pangu import spacing
```

Comment:
why not lazy import pangu and other lang specific utilities as well then?

Reply:
good point, it did not show up in the profiling, I can move it